tests/fixed_lines_01.cc

```diff
@@ -0,0 +1,48 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2014 by the deal.II authors
+//
+//    This file is subject to LGPL and may not be distributed
+//    without copyright and license information. Please refer
+//    to the file deal.II/doc/license.html for the  text  and
+//    further information on this license.
+//
+//-----------------------------------------------------------
+
+#include "tests.h"
+#include "utilities.h"
+#include <stdlib.h>
+#include <iostream>
+
+#include <iomanip>
+
+int main ()
+{
+
+  TimeUtilities tu;
+
+  unsigned int  N = 3;
+  int           a = 100;
+  double        b = 3.14;
+
+  OverWriteStream<std::ostream> out(std::cout, N);
```

Comment:
With the default value above, this should become

`OverWriteStream<> out(N);`

For this to work, the default value of `std::cout` has to be the second argument, not the first.